### ISSUE TITLE
Wrong position for U+0740 Syriac Feminine Dot

### ISSUE THREAD

**Issue by brawer** [googlei18n/noto-fonts#710]
U+0740 SYRIAC FEMININE DOT should be positioned high up. Currently it’s at the baseline, so it looks like a full stop.

**Comment by jungshik** [comment]
@kmansourMT : can you take a look?

**Comment by kmansourMT** [comment]
u+0740 is a mark that usually follows the Taw consonant (u+072C) to indicate that the word is feminine. In the Estrangelo style, it sits on the upper left side. Following is the combination as I see it rendered. Please indicate what error you see.

<img width="94" alt="screen shot 2016-06-06 at 10 21 23" src="https://cloud.githubusercontent.com/assets/12738181/15831157/0a979af2-2bd1-11e6-84a1-6fcb19b29f7f.png">

**Comment by jungshik** [comment]
@kmansourMT Thank you for the prompt reply. Back to @brawer for more information.

**Comment by jungshik** [comment]
Back to @kmansourMT 

Relaying what Ned @ apple received from a user:
------------------
These two strings appear nearly identical

harmless2.jpg.app - using the normal ASCII character U+002E FULL STOP
harmless2 ݀jpg.app - using U+0740 SYRIAC FEMININE DOT to spoof the extension

According to http://www.unicode.org/Public/security/latest/ this character (U+0740 SYRIAC FEMININE DOT) resembles U+0307 COMBINING DOT ABOVE:

0740 ;  0307 ;  MA  # ( ݀ → ̇ ) SYRIAC FEMININE DOT → COMBINING DOT ABOVE # →݁→

but at least on OS X, it does not appear to resemble U+0307 COMBINING DOT ABOVE.
-------------------

In other words: it would be nice if *the nominal glyph* was not so close to the baseline, at least when isolated.

**Comment by kmansourMT** [comment]
Although I find the basis of this complaint extremely dubious, I would like to examine the graphic evidence. What font are we speaking of—there are 3 Syriac styles? Please pass through the graphic file harmless2.jpg.

The information provided by http://www.unicode.org/Public/security... serves as a warning about potential compromises of security, but it does not prohibit their use. For example, the following 2 characters, Ϋ and Ÿ, look identical, but one represents Latin u+0178, the other Greek u+03AB. No one would call for one of these characters to change because they are so similar, in fact, identical. And, in fact, we never stop using them.

**Comment by brawer** [comment]
Received this picture from Apple:

![harmless2-jpg-app](https://cloud.githubusercontent.com/assets/1527880/16265155/0208bbc8-387e-11e6-866e-2dae533be5b1.png)

**Comment by verdy-p** [comment]
I also agree that in fact ALL combining dots in Syriac are at wrong positions. In the Unicode Chats and including in the character names, there is a clear indication that they should be either well above the baseline, or clearly below it (like descenders). For now they have all been set between the baseline and the x-height, they are ALL incorrect and ALL confusable.
It seems that these glyphs were just drawn rapidly, but never positioned (GPOS) correctly and accurately (and never tested in, combination with the base characters.
Their metrics are also incorrect (for proper display with a leading space). Their default position and advance width should also be tuned for the dotted circle glyph (when combining marks are shown in isolation, i.e. starting "defective" combining sequences).
